<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre1</class>
 <widget class="QMainWindow" name="fenetre1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1414</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bouton_fenetre1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>go</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxVache">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Nombres Vaches</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="button_vache_choix2">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_vache_choix3">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_vache_choix1">
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxHumain">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Nombres  humains</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="button_humain_choix3">
       <property name="text">
        <string>2000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_humain_choix1">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_humain_choix2">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_Conifere">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Nombres Conif√®res</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="button_conifere_choix1">
       <property name="text">
        <string>8000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_conifere_choix2">
       <property name="text">
        <string>10000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_conifere_choix3">
       <property name="text">
        <string>20000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Titre_page">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hands on earth</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Conifere_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Nombres algues</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="button_conifere_choix1_2">
       <property name="text">
        <string>40000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_conifere_choix2_2">
       <property name="text">
        <string>80000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_conifere_choix3_2">
       <property name="text">
        <string>100000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="description_jeu">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>Description jeu :</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_fond">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>image_fond</zorder>
   <zorder>bouton_fenetre1</zorder>
   <zorder>groupBoxVache</zorder>
   <zorder>groupBoxHumain</zorder>
   <zorder>groupBox_Conifere</zorder>
   <zorder>Titre_page</zorder>
   <zorder>groupBox_Conifere_2</zorder>
   <zorder>description_jeu</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
